src/WebJobs.Script/Description/EventHubBindingMetadata.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.ServiceBus;
+using System;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    public class EventHubBindingMetadata : BindingMetadata
+    {
+        [AllowNameResolution]
+        public string ConnectionString { get; set; }
+
+        [AllowNameResolution]
+        public string Path { get; set; }
+
+        public override void ApplyToConfig(JobHostConfigurationBuilder configBuilder)
+        {
+            EventHubConfiguration eventHubConfig = configBuilder.EventHubConfiguration;
+
+            if (this.IsTrigger)
+            {
+                string eventProcessorHostName = Guid.NewGuid().ToString();
+
+                string storageConnectionString = configBuilder.Config.StorageConnectionString;
+
+                var eventProcessorHost = new Microsoft.ServiceBus.Messaging.EventProcessorHost(
+                     eventProcessorHostName,
+                     this.Path,
+                     Microsoft.ServiceBus.Messaging.EventHubConsumerGroup.DefaultGroupName,
+                     this.ConnectionString,
+                     storageConnectionString);
+
+                eventHubConfig.AddEventProcessorHost(this.Path, eventProcessorHost);
```

Comment:
This is the glue from config in Function.Json to calling EventHubConfiguration. 

---
In reply to: [54124091](https://github.com/Azure/azure-webjobs-sdk-script/pull/26#discussion_r54124091) [](ancestors = 54124091)